pkg/controller/nebulabackup/nebula_backup_control.go

```diff
@@ -43,49 +44,39 @@ func NewBackupControl(clientSet kube.ClientSet, backupManager Manager) ControlIn
 	}
 }
 
-func (c *defaultBackupControl) SyncNebulaBackup(bp *v1alpha1.NebulaBackup) error {
-	phase := bp.Status.Phase
-	if phase == "" {
-		phase = v1alpha1.BackupPending
+func (c *defaultBackupControl) SyncNebulaBackup(bp *v1alpha1.NebulaBackup) (*time.Duration, error) {
```

Comment:
Few suggestions:
1. If certain conditions are met, you can end the sync and return directly, like BackupFailed, BackupInvalid, BackupComplete
2. Otherwise continue deal with the sync logic and update status
3. nextReconcile is conflict with the reconcileTimeOut, and nextReconcile is not needed here

Reply:
Fixed